pkgs/development/tools/database/liquibase/default.nix

```diff
@@ -63,12 +66,12 @@ stdenv.mkDerivation rec {
       #!/usr/bin/env bash
       # taken from the executable script in the source
       CP=""
+      ${addJars "$out/internal/lib"}
       ${addJars "$out/lib"}
       ${addJars "$out"}
       ${lib.concatStringsSep "\n" (map (p: addJars "${p}/share/java") extraJars)}
-
       ${lib.getBin jre}/bin/java -cp "\$CP" \$JAVA_OPTS \
-        liquibase.integration.commandline.LiquibaseCommandLine \''${1+"\$@"}
+      liquibase.integration.commandline.LiquibaseCommandLine \''${1+"\$@"}
```

Comment:
@ptliu-awake Is this the correct invocation?

Reply:
The original indentation produced 3 spaces in the run script as opposed to 1.

Before:

```bash
/nix/store/dvad9y3g9sd4kw3l2i1qk5s7dsg6w009-graalvm11-ce-22.3.1/bin/java -cp "$CP" $JAVA_OPTS   liquibase.integration.commandline.LiquibaseCommandLine ${1+"$@"}
```

After:

```bash
/nix/store/dvad9y3g9sd4kw3l2i1qk5s7dsg6w009-graalvm11-ce-22.3.1/bin/java -cp "$CP" $JAVA_OPTS liquibase.integration.commandline.LiquibaseCommandLine ${1+"$@"}
```